### ISSUE TITLE
buildah bud --target functionality missing

### ISSUE THREAD

**Issue by TomSweeneyRedHat** [containers/buildah#632]
**Description**
`buildah bud --target` is missing and needs to be provided.  This is functionality that Docker has.

#599 is the parent issue for this issue.

**Output of `buildah version`:**

Buildah v0.16

**Comment by rhatdan** [comment]
@TomSweeneyRedHat Now since @umohnani8 has added layers, this should be doable?  @umohnani8 WDYT?

**Comment by rhatdan** [comment]
Specifying target build stage (--target)

When building a Dockerfile with multiple build stages, --target can be used to specify an intermediate build stage by name as a final stage for the resulting image. Commands after the target stage will be skipped.

```
FROM debian AS build-env
...

FROM alpine AS production-env
...

$ docker build -t mybuildimage --target build-env .
```

**Comment by rhatdan** [comment]
@TomSweeneyRedHat Do we still have this issue?

**Comment by TomSweeneyRedHat** [comment]
@rhatdan I think it's now doable with @umohnani8 's changes for layering, but I don't think we've the code in place in the bud command to accept the --target option yet.

**Comment by h-vetinari** [comment]
Hitting this issue hard as I need multistage builds with `--target`, but can't upgrade docker from within RHEL past 1.13, and podman/buildah in the newest RHEL7.6 (or 8beta for that matter), do not yet replicate the 1.5year old ability of docker...

**Comment by rhatdan** [comment]
podman 1.0 and buildah 1.6 should have support for this.

**Comment by rhatdan** [comment]
Ok, could you make this happen.

**Comment by h-vetinari** [comment]
This issue can probably be closed now? It seems to have made it to 1.7 (at least according to the release notes).
